docs/developers/devel-env.rst

```diff
@@ -19,7 +19,7 @@ Clone listenbrainz-server
 -------------------------
 
 ListenBrainz is hosted on GitHub at https://github.com/metabrainz/listenbrainz-server/.
-You can use ``git`` to clone it to your computer
+You can use ``git`` to clone it (or your own fork/branch) to your computer
```

Comment:
```suggestion
You can use ``git`` to clone it (or your own fork) to your computer
```
Just a small technical correction here - you are cloning a fork, but not a branch